src/support/kuzzle.ts

```diff
@@ -88,11 +98,51 @@ export class KuzzleSDK {
           await this.sdk.auth.refreshToken()
         }
         catch (error) {
-          log(`Cannot refresh token: ${error}`)
+          logger.logKo(`Cannot refresh token: ${error.message}`)
         }
       }, 80 * SECOND)
 
-      log(chalk.green(`[ℹ] Loggued as ${this.username}.`))
+      logger.logInfo(`Loggued as ${this.username}.`)
+    }
+  }
+
+  /**
+   * Impersonates a user.
+   * Every action called in the given callback will be impersonated.
+   *
+   * @param {string} userKuid - User kuid to impersonate
+   * @param {Function} callback - Callback that will be impersonated
+   */
+  public async impersonate(userKuid: string, callback: Function) {
+    const currentToken = this.sdk.jwt
+
+    let apiKey: any
+
+    try {
+      const apiKey = await this.security.createApiKey(
+        userKuid,
+        'Kourou impersonation token',
+        { expiresIn: '1d', refresh: false })
```

Comment:
1d seems a bit too large to my taste... What about 1h instead?

Reply:
Actually I don't think it really matter because the API key is deleted.  
I put `1d` in the case of a user running a long command like export or import but I don't think they will impersonate someone.

I will put 2h